docs/testing-contracts.rst

```diff
@@ -5,13 +5,13 @@ Testing a Contract
 
 For testing Vyper contracts we recommend the use of `pytest <https://docs.pytest.org/en/latest/contents.html>`_ along with one of the following packages:
 
+    * `Titanoboa <https://github.com/vyperlang/titanoboa>`_: A Vyper interpreter, pretty tracebacks, forking, debugging and deployment features. Maintained by the Vyper team.
     * `Brownie <https://github.com/iamdefinitelyahuman/brownie>`_: A development and testing framework for smart contracts targeting the Ethereum Virtual Machine
-    * `Ethereum Tester <https://github.com/ethereum/eth-tester>`_: A tool suite for testing ethereum applications
 
 Example usage for each package is provided in the sections listed below.
 
 .. toctree::
     :maxdepth: 2
 
+    testing-contracts-titanoboa.rst
     testing-contracts-brownie.rst
```

Comment:
We can write something to replace, or it might be better to just link out to our docs (so this doesn't have to be maintained)

Reply:
Wonder if even the brownie one is still correct...